### PR TITLE
Remove roles related to remote execution feature from user test cases

### PR DESCRIPTION
**Pull request by oshtaier**
`Remote Execution` is optional plug-in for Satellite application, so it doesn't seem so critical to support testing roles for that feature
```
nosetests tests/foreman/ui/test_user.py -m all_roles
..
----------------------------------------------------------------------
Ran 2 tests in 331.002s

OK
```

### CODE REVIEW

File: robottelo/constants.py [hunk 1]
```diff
@@ -797,8 +797,6 @@
     'Edit hosts',
     'Edit partition tables',
     'Red Hat Access Logs',
-    'Remote Execution Manager',
-    'Remote Execution User',
```
Comment (sthirugn) [inline robottelo/constants.py]: For GA, Remote execution plugin will be installed by default.  If these roles are not present by default, this is perhaps a bug.

Comment (oshtaier) [inline robottelo/constants.py]: Hmm, I can see interesting difference for RHEL 6 and 7.
RHEL7:
image
RHEL6:
image

Is it intended behaviour?

Comment (abalakh) [inline robottelo/constants.py]: I think so, according to [BZ1322392](https://bugzilla.redhat.com/show_bug.cgi?id=1322392).

**Comment by sthirugn** [comment]
Yeah good point, is it possible to remove these only for rhel6?

**Comment by omaciel** [comment]
Minor but important correction: `Remote Execution` is not an optional feature for a Satellite system. I know that for RHEL 6 we're having it disabled but based on recent discussions, this feature has to be enabled on RHEL 6 and 7. So we do want to include them in, and sadly, they will fail on RHEL 6 for the time being.

**Comment by oshtaier** [comment]
(self-remainder :)) Look whether plugin is present in the system and then decide what roles to use

**Comment by oshtaier** [comment]
RHEL6:
```
nosetests tests/foreman/ui/test_user.py -m all_roles
..
----------------------------------------------------------------------
Ran 2 tests in 640.415s

OK
```

RHEL7:
```
[oshtaier@centosclient robottelo]$ nosetests tests/foreman/ui/test_user.py -m all_roles
..
----------------------------------------------------------------------
Ran 2 tests in 691.892s

OK
```

File: tests/foreman/ui/test_user.py [hunk 2]
```diff
@@ -40,6 +41,17 @@ def valid_strings(len1=10):
 class UserTestCase(UITestCase):
     """Implements Users tests in UI"""
 
+    @classmethod
+    def setUpClass(cls):  # noqa
```
Comment (abalakh) [inline tests/foreman/ui/test_user.py]: `# noqa` for what reason?

**Comment by abalakh** [comment]
ACK

**Comment by elyezer** [comment]
ACK

**Pull request merged**